app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -268,12 +266,18 @@ private void updateMenu() {
         final BrowserFragment browserFragment = getVisibleBrowserFragment();
         final boolean hasLoadedPage = browserFragment != null && !browserFragment.isLoading();
         final boolean canGoForward = browserFragment != null && browserFragment.canGoForward();
+
+        menu.findViewById(R.id.menu_turbomode).setSelected(isTurboEnabled());
```

Comment:
We don't need to do this here, let's move this to onSharedPreferenceChanged() since updateMenu is called everytime a page load is finished or menu is opened, that's an overkill for this situation.